cluster/lock.go

```diff
@@ -93,6 +93,11 @@ func (l *Lock) UnmarshalJSON(data []byte) error {
 		if err != nil {
 			return err
 		}
+	case isAnyVersion(version.Definition.Version, v1_5):
```

Comment:
no need for this case since you can just add v1_5 to the case above. We haven't changed anything in cluster-lock structure

Reply:
+1